### PR TITLE
DOC-718: Document new notification events

### PR DESCRIPTION
**Pull request by JamesToohey**
**Related Ticket:** 
DOC-718

**Description of Changes:**
Document new notification events

**Pre-checks:**
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~~`_data/nav.yml` has been updated (if applicable)~~
- [x] ~~Files has been included where required (if applicable)~~
- [x] ~~Files removed have been deleted, not just excluded from the build (if applicable)~~
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes56.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ For information on the `emojiimages` database and `emoticons_database` option, s
 
 For information on the `emoticons_images_url` option, see: [Emoticons plugin - `emoticons_images_url`]({{ site.baseurl }}/plugins/emoticons/#emoticons_images_url).
 
+### New Notification events
```
Comment (lnewson) [inline release-notes/release-notes56.md:72]: I don't think we need 2 release notes for the new events. Could we merge them into a single "New editor events" note instead please?

**Review by ashfordneil — APPROVED** [review]

File: advanced/events.md [hunk 2]
```diff
@@ -150,6 +150,8 @@ The following events are provided by the {{ site.productname }} editor.
...
 | Load                   | N/A                                                                                                                     | (iframe mode only) Fired when the editor iframe content has finished loading.                                                                                              |
 | ResolveName            | `{ name: string, target: Element }`                                                                                     | (iframe mode only) Fired when the editor tries to resolve the name of an element to render the status bar path.                                                            |
+| BeforeOpenNotification | `{ notification: NotificationSpec }`                                                                                    | Fired before a notification is displayed, see [`NotificationSpec`]({{site.baseurl}}/advanced/creating-custom-notifications/). {{site.requires_5_6v}}                       |
```
Comment (tylerkelly13) [inline advanced/events.md:153]:
```suggestion
| BeforeOpenNotification | `{ notification: NotificationSpec }`                                                                                    | Fired before a notification is displayed. For information on the Notifications Specification (`NotificationSpec`), see: [Create custom notifications]({{site.baseurl}}/advanced/creating-custom-notifications/). {{site.requires_5_6v}}                       |
```

File: advanced/events.md [hunk 3]
```diff
@@ -150,6 +150,8 @@ The following events are provided by the {{ site.productname }} editor.
...
+| OpenNotification       | `{ notification: NotificationSpec }`                                                                                    | Fired after a notification is displayed, see [`NotificationSpec`]({{site.baseurl}}/advanced/creating-custom-notifications/). {{site.requires_5_6v}}                        |
```
Comment (tylerkelly13) [inline advanced/events.md:154]:
```suggestion
| OpenNotification       | `{ notification: NotificationSpec }`                                                                                    | Fired after a notification is displayed. For information on the Notifications Specification (`NotificationSpec`), see: [Create custom notifications]({{site.baseurl}}/advanced/creating-custom-notifications/). {{site.requires_5_6v}}                        |
```

File: release-notes/release-notes56.md [hunk 4]
```diff
@@ -49,11 +49,14 @@ A new optional `name` field that sets a specific name to the format when it's be
...
 
-### New `TableModified` event
+### New editor events
 
-The new `TableModified` event is fired when style or structural changes are made to a table.
+- The new `TableModified` event is fired when style or structural changes are made to a table.
+- Two new core events that are fired before and after displaying notifications.
 
-For information on the `TableModified` event, see: [Table plugin - `events`]({{ site.baseurl }}/plugins/table/#events).
+For more information on the `TableModified` event, see: [Table plugin - `events`]({{ site.baseurl }}/plugins/table/#events).
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:57]:
```suggestion
For information on the `TableModified` event, see: [Table plugin - `events`]({{ site.baseurl }}/plugins/table/#events).
```

File: release-notes/release-notes56.md [hunk 5]
```diff
@@ -49,11 +49,14 @@ A new optional `name` field that sets a specific name to the format when it's be
...
+
+For more information on core editor events, see: [Editor core events]({{ site.baseurl }}/advanced/events/#editorcoreevents).
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:59]:
```suggestion
For information on core editor events, see: [Editor core events]({{ site.baseurl }}/advanced/events/#editorcoreevents).
```

File: release-notes/release-notes56.md [hunk 6]
```diff
@@ -49,11 +49,14 @@ A new optional `name` field that sets a specific name to the format when it's be
 
 For information on the `name` field, see: [User interface - `style_formats`]({{ site.baseurl }}/configure/editor-appearance/#style_formats).
 
-### New `TableModified` event
+### New editor events
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:53]:
```suggestion

{{site.productname}} 5.6 introduces several new events that integrators can use for custom functionality:

```

File: release-notes/release-notes56.md [hunk 7]
```diff
@@ -49,11 +49,14 @@ A new optional `name` field that sets a specific name to the format when it's be
...
 
-The new `TableModified` event is fired when style or structural changes are made to a table.
+- The new `TableModified` event is fired when style or structural changes are made to a table.
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:54]:
```suggestion
- A new `TableModified` event which is fired when a style or structural change is made to a table.
```

File: release-notes/release-notes56.md [hunk 8]
```diff
@@ -49,11 +49,14 @@ A new optional `name` field that sets a specific name to the format when it's be
...
+- Two new core events that are fired before and after displaying notifications.
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:55]:
```suggestion
- Two new core events which are fired before and after displaying notifications.
```

**Review by tylerkelly13 — APPROVED** [review]
LGTM, Just a few suggested changes

**Review by EkimChau — APPROVED** [review]

**Pull request merged**